modules/HardwareDrivers/PowerMeters/CarloGavazzi_DCT1/meter/base64.hpp

```diff
@@ -0,0 +1,539 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
```

Comment:
This file looks like [base64.hpp](https://github.com/tobiaslocker/base64/blob/master/include/base64.hpp) from https://github.com/tobiaslocker/base64. If so, the license is MIT, and this should be reflected in the header.